src/Sources/LightPortal/Addons/ConfigProvider.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bugo\LightPortal\Addons;
+
+use Bugo\LightPortal\Addons\AddonManager;
+use Bugo\LightPortal\Addons\AddonManagerFactory;
+use Laminas\ServiceManager\Factory\InvokableFactory;
+
+final class ConfigProvider
+{
+    public function __invoke(): array
+    {
+        return [
+            'dependencies' => $this->getDependencies(),
+            'lp_addons'    => $this->getAddonsConfig(),
+        ];
+    }
+
+    /**
+     * Configuration for ServiceManager for the AddonManager
+     * @return array
+     */
+    public function getDependencies(): array
+    {
+        return [
+            'aliases'      => [],
+            'factories'    => [
+                AddonManager::class => AddonManagerFactory::class
+            ],
+            'delegators'   => [],
+        ];
+    }
+
+    /**
+     * Default configuration to seed the AddonManager
+     * @return array
+     */
+    public function getAddonsConfig(): array
+    {
+        return [
+            'initializers' => [],
+            'aliases'      => [
+				'userInfo' => Addons\UserInfo\UserInfo::class,
```

Comment:
I need snake_case here: `'user_info' => Addons\UserInfo\UserInfo::class`

Reply:
Well then all I have to do is create the service Alias as user_info ;)